### DISCUSSION TITLE
How can I find names of buttons?

### DISCUSSION THREAD

**Discussion by DAT4** [hyprwm/Hyprland#1591] [Q&A] [answered]
I don't see anywhere how you get the names of buttons for the config.

For instance I would like to map SUPER + COLON to moveworkspace, prev but when I write COLON how I would do in dwm (i think) then nothing happens. 

How do I get the name of the button? 

is it related to xkb or some specific waylandkb thingy ? :) 

Thank you :)

**Comment by vaxerski** [comment, marked as answer]
https://wiki.hyprland.org/Configuring/Binds/#uncommon-syms--binding-with-a-keycode

No way, wiki mentions that

**Reply by DAT4** [reply]
Thanks - I almost got it right. I just need lowercase :D
